data/tribes/initialization/atlanteans/helptexts.lua

```diff
@@ -0,0 +1,927 @@
+-- Atlantean helptexts
+
+-- For formatting time strings
+include "tribes/scripting/help/global_helptexts.lua"
+
+-- RST
+-- .. _lua_tribes_tribes_helptexts:
+--
+-- Helptexts
+-- ---------
+--
+-- Helptexts are defined in ``tribes/initialization/<tribename>/helptexts.lua`` and linked to from :ref:`units.lua <lua_tribes_tribes_units>`.
+--
+--
+-- You will need to return a table of helptexts by map object type and category. It looks like this:
+--
+-- .. code-block:: lua
+--
+--    push_textdomain("tribes_encyclopedia")
+--    local result = {
+--       buildings {
+--          barbarians_ax_workshop = {
+--             lore = {
+--                -- TRANSLATORS: Lore helptext for a barbarian production site: Ax Workshop
+--                pgettext("barbarians_building", "‘A new warrior’s ax brings forth the best in its wielder – or the worst in its maker.’")
```

Comment:
This syntax is a bit awkward. `lore = pgettext("foo", "bar"),` would be more readable to me. How about allowing plain strings as values as well as those tables, at the scripter's choice?

Reply:
That sounds doable.